<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-26T02:48:18 EDT" name="Suite1" finished-at="2023-05-26T02:48:25 EDT" duration-ms="6938">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:smarttech.ny.testrunner.MyRunner@27462a88]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-05-26T02:48:18 EDT" name="functionaltest" finished-at="2023-05-26T02:48:25 EDT" duration-ms="6938">
      <class name="smarttech.ny.testrunner.MyRunner">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:smarttech.ny.testrunner.MyRunner@27462a88]" started-at="2023-05-26T02:48:18 EDT" name="setUpClass" finished-at="2023-05-26T02:48:18 EDT" duration-ms="275" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:smarttech.ny.testrunner.MyRunner@27462a88]" started-at="2023-05-26T02:48:18 EDT" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-26T02:48:25 EDT" duration-ms="6575" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["User Able To Login, Purchase And Download Invoice."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Valid Users Able To Download Invoice after purchase the order.]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home] but found [expected]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home] but found [expected]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at smarttech.ny.stepdefinitions.FunctionalStepDef.verify_that_home_page_is_visible_successfully(FunctionalStepDef.java:29)
at âœ½.Verify that home page is visible successfully(file:///C:/Users/nathb/eclipse-workspace/Automation_Excercising_E2E/./Features/functionalTest.feature:6)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:smarttech.ny.testrunner.MyRunner@27462a88]" started-at="2023-05-26T02:48:25 EDT" name="tearDownClass" finished-at="2023-05-26T02:48:25 EDT" duration-ms="63" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- smarttech.ny.testrunner.MyRunner -->
    </test> <!-- functionaltest -->
  </suite> <!-- Suite1 -->
</testng-results>
